src/Core/Meeting.php

```diff
@@ -379,6 +379,36 @@ public function getAttendees()
         return $this->attendees;
     }
 
+    /**
+     * Moderators of Meeting - Subset of Attendees
+     * @return Attendee[]
+     */
+    public function getModerators()
```

Comment:
```suggestion
    public function getModerators(): array
```

I guess we should add return type hints when pulling upstream changes and their miss them. As our library has an independent release schedule and you must decide to use it, I guess that is not a big problem.